torch/onnx/symbolic_opset9.py

```diff
@@ -2311,11 +2311,12 @@ def _var_mean(g, input, dim, unbiased, keepdim):
         num_elements = numel(g, input)
     else:
         mean = g.op("ReduceMean", input, axes_i=dim, keepdims_i=keepdim)
+        t_mean = g.op("ReduceMean", input, axes_i=dim, keepdims_i=1)
```

Comment:
Don't quite get the logic here why this change can resolve this problem. The only change here is for `keepdims`, seems not change the logic very much...

Reply:
For example ->
if the input shape is (x,y,z) and dim values are (1,2) and keepdims is set to 0, the resulting shape of the reduceMean node is (x). Now when this node is passed to the Sub op, the multi-directional broadcasting converts this shape to (1,1,x). 
So essentially we are subtracting shapes (x,y,z) and (1,1,x). Hence we are setting keepdims to 1 always to avoid this mismatch in dimensions for the sub op.
But for cases in which user has passed keepdims=0, the mean value would be of wrong dimensions, hence the difference in t_mean and mean.